paylike.php

```diff
@@ -1,65 +1,88 @@
 <?php
-
-class paylike extends PaymentModule {
```

Comment:
As I am extending paylike from PaymentModule prestashop core class and I don't need to commit it. You can find PaymentModule.php class in "prestashop_root_dir/classes" folder.